capsules/src/symmetric_encryption.rs

```diff
@@ -0,0 +1,310 @@
+//! Symmetric Block Cipher Capsule
+//!
+//! Provides a simple driver for userspace applications to encrypt and decrypt messages
+//!
+//! The key is assumed to be 16, 24 or 32 bytes
+//!
+//! Author: Niklas Adolfsson <niklasadolfsson1@gmail.com>
+//! Author: Fredrik Nilsson <frednils@student.chalmers.se>
+//! Date: March 16, 2017
```

Comment:
Comments for a driver should include a list with explanation of the available command, allow and subscribe syscalls as well as summary of the basic expected operation from userland. The new CRC driver is basically the only existing driver that does a good job of this so probably use that as an example.